src/clients/createSmartAccountClient.ts

```diff
@@ -16,6 +16,11 @@ import {
     smartAccountActions
 } from "./decorators/smartAccount.js"
 
+/**
+ * TODO:
+ *  - Add docs
+ *  - Fix typing, 'accounts' is required to signMessage, signTypedData, signTransaction, but not needed here, since account is embedded in the client
```

Comment:
This is not needed, as in viem in `signTransaction` you can have a client without an account and attach the client to any of the above function calls separately.